### PR TITLE
windows: Support GCE

### PR DESCRIPTION
**Pull request by lmm**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: scripts/install-calico-windows.ps1 [hunk 1]
```diff
@@ -125,6 +125,17 @@ function GetPlatformType()
...
 
+    # GCE
+    $restError = $null
+    Try {
+        $gceNodeName = Invoke-RestMethod -UseBasicParsing -Headers @{"Metadata-Flavor"="Google"} "http://metadata.google.internal/computeMetadata/v1/instance/hostname" -ErrorAction Ignore
+    } Catch {
+        $restError = $_
+    }
+    if ($restError -eq $null) {
+        return ("ec2")
```
Comment (song-jiang) [inline scripts/install-calico-windows.ps1:136]:
```suggestion
        return ("gce")
```

**Review by song-jiang — CHANGES_REQUESTED** [review]
One typo

File: scripts/install-calico-windows.ps1 (hunk 1 above)
Comment (lmm) [reply scripts/install-calico-windows.ps1:136]: sigh. thanks for catching that

**Review by song-jiang — APPROVED** [review]
LGTM

**Pull request merged**